### ISSUE TITLE
Consolidate the same constants definitions into one file.

### ISSUE THREAD

**Issue by KentaroV2** [KentaroV2/excel-first#12]
Consolidate the same constants definitions into one file for easy maintenance.
Here is an example of some constants:

```vb
Private Const first_Level_Delimiter As String = vbTab
Private Const exit_Status__Success As Long = 0
```

Options are:
1. Newly create “CommonConstants.bas” file.
2. Define the same constants in a certain file(.bas).

**Comment by KentaroV2** [comment]
I recommend that the module name is **CommonDefinitions** since it's file may include:
- public constant, public variables, and
- declaration of windows functions.

```vb
Public Const first_Level_Delimiter As String = vbTab
Public Const exit_Status__Success As Long = 0
Declare Sub Sleep Lib "kernel32" (ByVal dwMilliseconds As Long)
```

**Comment by KentaroV2** [comment]
This issue is closed with merging feature/issue-#15 branch into develop branch.
